lib/linguist/heuristics.rb

```diff
@@ -274,7 +274,7 @@ def call(data)
     end
 
     disambiguate ".pl" do |data|
-      if /^(use v6|(my )?class|module)/.match(data)
+      if /^(use v6|(my)? class|module)/.match(data) #class might be a variable name in Perl 5
```

Comment:
Could you add a test for this? I don't really understand the reasoning behind this :/